pkg/worker/handler/endpoint/ensure.go

```diff
@@ -0,0 +1,37 @@
+package endpoint
+
+import (
+	"net/http"
+
+	"github.com/xh3b4sd/tracer"
+)
+
+func (h *Handler) Ensure() error {
+	var err error
+
+	var res *http.Response
+	{
+		res, err = http.Get(h.end)
+		if err != nil {
+			return tracer.Mask(err)
+		}
+	}
+
+	{
+		defer res.Body.Close()
+	}
+
+	var hlt float64
+	if res.StatusCode == http.StatusOK {
+		hlt = 1
+	}
```

Comment:
If we get a status code `200`, then the service is considered healthy by setting the data point to `1`, otherwise `0`.

Reply:
Yes, see https://go.dev/tour/basics/12. The zero values of complex types like pointers, channels and slices is `nil`.